src/content/en/tools/workbox/modules/workbox-webpack-plugin.md

```diff
@@ -0,0 +1,86 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: The module guide for workbox-webpack-plugin.
+
+{# wf_updated_on: 2017-12-15 #}
+{# wf_published_on: 2017-12-15 #}
+{# wf_blink_components: Blink>ServiceWorker #}
+
+# Workbox webpack Plugin  {: .page-title }
+
+## What's the Workbox webpack Plugin?
+
+This is a plugin for the [webpack](https://webpack.js.org/) which generates a list of URLs to
+[precache](workbox-precaching) (known as a "precache manifest") based on the assets in a webpack
```

Comment:
I wouldn't link this up to workbox-precaching. I would expect this to go to a page that is about what the term "precache means" rather than a guide on another module.

Reply:
I've changed the link to point to `/web/tools/workbox/guides/precache-files`.